src/js/components/Form/Form.js

```diff
@@ -655,9 +657,11 @@ const Form = forwardRef(
           }
         }}
       >
-        <FormContext.Provider value={formContextValue}>
-          {children}
-        </FormContext.Provider>
+        <FormTypeContext.Provider value={kind}>
+          <FormContext.Provider value={formContextValue}>
```

Comment:
Should we just add `kind` to the FormContext? Instead of making its own

Reply:
yeah I tried this and ran into issues... I couldn't figure some of it out I can take another look though